packages/@aws-cdk/aws-apigatewayv2-authorizers/README.md

```diff
@@ -256,3 +257,24 @@ new apigwv2.WebSocketApi(this, 'WebSocketApi', {
   },
 });
 ```
+
+### IAM Authorizer
```

Comment:
There is already a section in this README with this name.

Reply:
Sorry, should have made it more obvious. The other one is under `HTTP` API's and this one is under `WebSocket` and uses the singular word "Authorizer". I can make them separate but thought I'll copy the style that's been followed 😄